templates/org/aac/aac-architecture-guide-content-browse.md

```diff
@@ -1,47 +1,63 @@
-Architecture Guides don't have specific content guidelines (except for the three sections at the end), but they should follow all Microsoft and Learn style and pull request criteria.
```

Comment:
@ckittel - Let me know if you'd like me to omit this file from the PR. I think you mentioned that architecture guides generally shouldn't use the browser pattern.

Reply:
Yeah, I'd say if we can, to delete the "guide as a browse page" guidance in general -- and leave it as a non-browser version, that would be ideal.  If someone has a good reason to make the guide a browse page, we can make it a one-off and help.  You don't need to do the "purge" as part of this PR, but if you want to skip this file, that would be fine.